File: trunk/NAI/Surface/NAI/Communication/MessageLayer/Messages/Outgoing/OutgoingBodyLessMessage.cs

namespace NAI.Communication.MessageLayer.Messages.Outgoing
{
    internal abstract class OutgoingBodyLessMessage : OutgoingMessage
    {
        public OutgoingBodyLessMessage(MessageProtocol command)
            : base(command)
        { }

        protected override byte[] PrepareMessageBody()
        { 
            return new byte[0];
        }
    }
}

File: NAI/Android/src/dk/itu/nai/comm/messages/outgoing/OutgoingBodyLessMessage.java
package dk.itu.nai.comm.messages.outgoing;

import java.io.IOException;

import dk.itu.nai.comm.Protocol;

public class OutgoingBodyLessMessage extends OutgoingMessage {

	public OutgoingBodyLessMessage(Protocol command) {
		super(command);
	}

	@Override
	protected byte[] prepareMessageBody() throws IOException {
		return new byte[0];
	}

	
}
